msvs-promote-path

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Hah - I've already pushed that to `master`!

Reply:
aah :D let me rebase